src/Sylius/Behat/Context/Api/Shop/ProductContext.php

```diff
@@ -178,6 +182,14 @@ public function itsCurrentVariantShouldBeNamed(string $variantName): void
         );
     }
 
+    /**
+     * @Then I should see empty list of products
+     */
+    public function iShouldSeeEmptyListOfProducts(): void
+    {
+        Assert::same(0, $this->responseChecker->countTotalCollectionItems($this->client->getLastResponse()));
```

Comment:
`Assert::same($value, $expect, $message = '')` 😄 
```suggestion
        Assert::same($this->responseChecker->countTotalCollectionItems($this->client->getLastResponse()), 0);
```